.github/workflows/mirror_gitee.yml

```diff
@@ -0,0 +1,18 @@
+name: Mirror to Gitee Repo
+
+on: [ push, delete, create ]
+
+# Ensures that only one mirror task will run at a time.
+concurrency:
+  group: git-mirror
+
+jobs:
+  git-mirror:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: wearerequired/git-mirror-action@v1
+        env:
+          SSH_PRIVATE_KEY: ${{ secrets.SYNC_GITEE_PRIVATE_KEY }}
+        with:
+          source-repo: "git@github.com:deepmodeling/dpdata.git"
```

Comment:
Have you added the ssh key to your GitHub account?

Reply:
Seems the action failed in deepks-kit.

Reply:
This key is added to my gitee account, and github organization.

Reply:
It worked with dpgen.

Reply:
I added "ORANIZATION: deepmodeling" under "env:" in mirror_gitee.yml in a recent commit.

Reply:
I mean you use SSH to visit GitHub (git@github.com), so you must add this key to someone's GitHub account.